pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -355,9 +355,6 @@ buildStdenv.mkDerivation ({
     configureFlagsArray+=("--with-mozilla-api-keyfile=$TMPDIR/mls-api-key")
   '' + lib.optionalString (enableOfficialBranding && !stdenv.is32bit) ''
     export MOZILLA_OFFICIAL=1
-  '' + lib.optionalString stdenv.hostPlatform.isMusl ''
-    # linking firefox hits the vm.max_map_count kernel limit with the default musl allocator
```

Comment:
have you tested firefox with musl now?

Reply:
Kinda thought that CI would do that automatically...
Any way to ask it to test manually?

Reply:
@ofborg build pkgsMusl.firefox

Reply:
`pkgsMusl.firefox`
the value change won't be applied on the builder so can't test that way

Reply:
CI is alas not panacea :D

Reply:
Maybe I'd better revert this line.
Or should I replace it with a TODO? ("default vm.max_map_count has been increased, retest without this")

Reply:
todo